atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -336,6 +336,9 @@ def get_odcs_config(self):
             odcs_config = ODCSConfig(**odcs_config_kwargs)
         return odcs_config
 
+    def is_default(self):
```

Comment:
@rcerven is investigating if we can simply drop this mocked class since it's pretty much a copy of `ReactorConfig`.

Reply:
ok done, there was problem with that modification of conf in resolve_compose test, which actually was affecting values in new ReactorConfig instance ...